### PR TITLE
Improve code snippets

### PR DESCRIPTION
**Pull request by shaurmanchic**
TODO:
- marry the .sh scripts and dockerfiles in bash, node and python to save the result in try-it-results.json as strings for inbound and outbound respectively for each language
- .sh scripts and dockerfiles were copied from the demo 'as is'

### CODE REVIEW

File: content-by-language/bash/outbound-integration.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 curl {VGS_SAMPLE_ECHO_SERVER}/post -k \\
   -x {ACCESS_CREDENTIALS}@{VAULT_HOST}:{PORT} \\
   -H "Content-type: application/json" \\
-  -d '{"account_number": "{ALIAS}"}'
+  -d '{"account_number": "tok_sandbox_oTktJmBaAhUyGHnNJt7WMA"}'
```
Comment (ruslanmalogulko) [inline content-by-language/bash/outbound-integration.sh:4]: Should it be hardcoded here?

File: content-by-language/java/outbound-integration.java [hunk 2]
```diff
@@ -10,55 +12,56 @@
...
+        .setDefaultCredentialsProvider(provider)
+        .setSSLContext(buildSSLContext())
+        .build();
 
-  HttpPost httpPost = new HttpPost("{VGS_SAMPLE_ECHO_SERVER}/post");
-  httpPost.setHeader("Content-Type", "application/json");
-  httpPost.setEntity(new StringEntity("{\"account_number\":\"{ALIAS}\"}"));
+    final HttpPost httpPost = new HttpPost("{VGS_SAMPLE_ECHO_SERVER}/post");
+    httpPost.setHeader("Content-Type", "application/json");
+    httpPost.setEntity(new StringEntity("{\"account_number\":\"tok_sandbox_oTktJmBaAhUyGHnNJt7WMA\"}"));
```
Comment (ruslanmalogulko) [inline content-by-language/java/outbound-integration.java:46]: should it be hardcoded here? I can add it at personalization step, so:
- for test user we would have this alias
- for logged in user would be `{ALIAS}` but with tooltip that here should be some redacted data

File: content-by-language/node/outbound-integration.js [hunk 3]
```diff
@@ -2,36 +2,30 @@ const fs = require('fs');
...
   let result;
-
   try {
-    result = await fetch('https://echo.apps.verygood.systems/post', {
+    result = await fetch('{VGS_SAMPLE_ECHO_SERVER}/post', {
       method: 'POST',
       headers: { 'Content-Type': 'application/json' },
       body: JSON.stringify({
-        account_number: '{ALIAS}',
+        account_number: 'tok_sandbox_w8CBfH8vyYL2xWSmMWe3Ds',
```
Comment (ruslanmalogulko) [inline content-by-language/node/outbound-integration.js:22]: and here

File: content-by-language/python/outbound-integration.py [hunk 4]
```diff
@@ -1,7 +1,8 @@
...
-response = requests.post("{VGS_SAMPLE_ECHO_SERVER}/post",
-                         json={'account_number': '{ALIAS}'},
-                         verify='/full/path/to/cert.pem')
-print(str(response.content))
+
+os.environ['HTTPS_PROXY'] = 'https://{ACCESS_CREDENTIALS}@{VAULT_HOST}:{PORT}'
+response = requests.post('{VGS_SAMPLE_ECHO_SERVER}/post',
+                         json={'account_number': 'tok_sandbox_w8CBfH8vyYL2xWSmMWe3Ds'},
+                         verify='/opt/app/cert.pem')
+print(str(response.json()))
```
Comment (ruslanmalogulko) [inline content-by-language/python/outbound-integration.py:8]: missing empty line EOF

File: content-by-language/ruby/inbound-integration.rb [hunk 5]
```diff
@@ -6,6 +6,6 @@
 http = Net::HTTP.new(uri.host, uri.port)
 http.use_ssl = true
 request = Net::HTTP::Post.new(uri.path, initheader = {'Content-Type' =>'application/json'})
-request.body = {account_number: 'account_value'}.to_json
+request.body = {account_number: 'ACC00000000000000000'}.to_json
 response = http.request(request)
-puts "Response #{response.code} #{response.message}: #{response.body}"
+puts "Response #{response.code} #{response.message}: #{response.body}"
```
Comment (ruslanmalogulko) [inline content-by-language/ruby/inbound-integration.rb:11]: missing empty line EOF

File: content-by-language/ruby/outbound-integration.rb [hunk 6]
```diff
@@ -3,15 +3,15 @@
...
+http.ca_file = '../../mixed-content/sandbox_cert.pem'
 http.verify_mode = OpenSSL::SSL::VERIFY_PEER
 http.verify_depth = 5
 
 request = Net::HTTP::Post.new(uri.path, initheader = {'Content-Type' =>'application/json'})
-request.body = {account_number: '{ALIAS}'}.to_json
+request.body = {account_number: 'tok_sandbox_w8CBfH8vyYL2xWSmMWe3Ds'}.to_json
 response = http.request(request)
-puts "Response #{response.code} #{response.message}: #{response.body}"
+puts "Response #{response.code} #{response.message}: #{response.body}"
```
Comment (ruslanmalogulko) [inline content-by-language/ruby/outbound-integration.rb:17]: missing empty line EOF

File: content-by-language/bash/outbound-integration.sh (hunk 1 above)
Comment (shaurmanchic) [reply content-by-language/bash/outbound-integration.sh]: oops

**Review by ruslanmalogulko — APPROVED** [review]

**Pull request merged**